src/main/java/pl/edu/agh/firecell/model/material/MaterialConductionMap.java

```diff
@@ -7,33 +7,31 @@
 
 public class MaterialConductionMap {
 
-    private final Map<Pair, Double> coefficientMap;
+    private final Map<UnorderedMaterialPair, Double> coefficientMap;
 
     public MaterialConductionMap() {
         coefficientMap = new HashMap<>();
     //==================================================================================================================
-        coefficientMap.put(new Pair(Material.AIR, Material.AIR),                                            /**/ 0.008);
-        coefficientMap.put(new Pair(Material.AIR, Material.WOOD),                                            /**/ 0.02);
-        coefficientMap.put(new Pair(Material.AIR, Material.CELLULAR_CONCRETE),                              /**/ 0.015);
+        coefficientMap.put(new UnorderedMaterialPair(Material.AIR, Material.AIR),                                            /**/ 0.008);
+        coefficientMap.put(new UnorderedMaterialPair(Material.AIR, Material.WOOD),                                            /**/ 0.02);
+        coefficientMap.put(new UnorderedMaterialPair(Material.AIR, Material.CELLULAR_CONCRETE),                              /**/ 0.015);
     //==================================================================================================================
-        coefficientMap.put(new Pair(Material.WOOD, Material.WOOD),                                            /**/ 0.2);
-        coefficientMap.put(new Pair(Material.WOOD, Material.CELLULAR_CONCRETE),                              /**/ 0.15);
+        coefficientMap.put(new UnorderedMaterialPair(Material.WOOD, Material.WOOD),                                            /**/ 0.2);
+        coefficientMap.put(new UnorderedMaterialPair(Material.WOOD, Material.CELLULAR_CONCRETE),                              /**/ 0.15);
     //==================================================================================================================
-        coefficientMap.put(new Pair(Material.CELLULAR_CONCRETE, Material.CELLULAR_CONCRETE),                  /**/ 0.1);
+        coefficientMap.put(new UnorderedMaterialPair(Material.CELLULAR_CONCRETE, Material.CELLULAR_CONCRETE),                  /**/ 0.1);
     //==================================================================================================================
         try {
             validate();
-        } catch (ConductionCoefficientException e) {
-            e.printStackTrace();
-        }
+        } catch (ConductionCoefficientException ignored) {}
+
     }
 
     public Double getCoefficient(Material m1, Material m2) {
```

Comment:
Make it return primitive `double` instead of `Double`. The wrapped double was needed only inside map.